price-feeder/oracle/provider/huobi.go

```diff
@@ -305,6 +302,16 @@ func (p *HuobiProvider) messageReceived(messageType int, bz []byte, reconnectTic
 			"provider",
 			config.ProviderHuobi,
 		)
+		return
+	}
+
+	if tickerErr != nil || candleErr != nil {
```

Comment:
remove `if` - always log

Reply:
okay, done at https://github.com/umee-network/umee/pull/684/commits/feca618d84f4657fc341b3fe99b7e01d56da3cbb
but [this](https://github.com/umee-network/umee/pull/684#discussion_r835300910) will keep happening
@toteki